content/general/concepts/full-stack/full-stack.md

```diff
@@ -0,0 +1,77 @@
+
+## Common Tasks
+
+Full-stack web development can involve a variety of front-end and back-end tasks, including:
+
+- Creating and updating webpages.
+- Making sure webpages are responsive and work well on phones or tablets.
+- Improving website performance.
+- Adding and improving accessibility features.
+- Using front-end frameworks to build client-side software.
+- Building and consuming [APIs](https://www.codecademy.com/resources/docs/general/api).
+- Creating, integrating, and managing databases.
+- Using back-end frameworks to build server-side software.
+- Validating data to make sure it's formatted correctly before being sent to the database.
+- Integrating user-facing elements with server-side elements to make sure that information is being sent to the right place so the server can retrieve it.
+
+## Popular Full-Stack Tech Stacks
+
+Full-stack developers typically utilize a combination of software, called a stack, in order to build full-stack applications.
+
+### MERN Stack
+
+A couple highlights for the MERN stack include the full-stack JavaScript nature and the fact that React is used, which is popular with both developers and designers.
+
+- [MongoDB](https://www.codecademy.com/learn/learn-mongodb): NoSQL database that uses [JSON](https://www.codecademy.com/resources/docs/general/json) (BSON)
+- [Express.js](https://www.codecademy.com/resources/docs/general/express): Back-end framework for Node.js applications.
+- [React.js](https://www.codecademy.com/learn/react-101): Front-end framework.
+- [Node.js](https://www.codecademy.com/resources/docs/general/node-js): JavaScript runtime for server-side applications.
+
+### MEAN Stack
+
+The MERN stack is popular for a number of reasons, a couple being that both the front-end and back-end are programmed in JavaScript and that the entire stack is open-source.
+
+- MongoDB: NoSQL database that uses JSON (BSON)
+- Express.js: Back-end framework for Node.js applications.
+- [Angular](https://www.codecademy.com/resources/docs/general/angular): Front-end framework.
+- Node.js: JavaScript runtime for server-side applications.
+
+### LAMP Stack
+
+Hightlights for the LAMP stack include the use of Linux as well as MySQL, two popular open-source projects.
+
+- [Linux](https://www.codecademy.com/resources/docs/general/linux): Open-source operating system.
+- Apache: Popular open-source web server software.
+- MySQL: Database management system.
+- [PHP](https://www.codecademy.com/learn/learn-php): Web development programming language.
+
+### Django Stack
+
+Highlights for this stack include Python's relative easiness to learn and Django's ability to simplify building templates.
```

Comment:
```suggestion
Highlights for this stack include the use of the Python, and the Django framework, a popular web development platform that is structured around creating templates.
```

Reply:
Optional rephrasing that just aims to simplify a bit. The beginner-friendly quality of Python is addressed twice in this section

Reply:
💯